src/components/QuickOptions.tsx

```diff
@@ -73,6 +77,10 @@ export function QuickOptions({
           handleClick={(event) =>
             handleQuickOption(event, QuickOption.Essential)
           }
+          ariaDescription={formatMessage(
+            quickOptionsMessages.buttonAriaDescription,
+          )}
```

Comment:
IMO, all these descriptions on each button make it really difficult for a SR to get through each button, especially since they are repetitive. I would opt for some kind of group role with a label, where the label has this description and it's clear that it applies to all the buttons.

Reply:
i agree actually, let me fix this up